app.organicmaps.desktop.yml

```diff
@@ -0,0 +1,50 @@
+app-id: app.organicmaps.desktop
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: OMaps-wrapper
+rename-desktop-file: OrganicMaps.desktop
+rename-icon: organicmaps
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+modules:
+  - name: rsync # used at build time
+    buildsystem: autotools
+    config-opts:
+      - --disable-md2man
+      - --disable-xxhash
+    sources:
+      - type: git
+        url: https://github.com/WayneD/rsync
+        tag: v3.2.3
+        commit: ec1abcfd29d98cc7a4864fd3a8ffc6245770ef0c
```

Comment:
for some reason the tag doesn't match the commit.

<img width="529" alt="image" src="https://user-images.githubusercontent.com/114441/162839846-ded66991-e650-4e1f-b596-2ea48b400642.png">

Reply:
ah it got changed your previous commit. It was correct before.

Reply:
Oops, I did that commit in a rush and changed the wrong hash. Fixed.

bot, build app.organicmaps.desktop